Clarify units and directional distortion of -simplify and -segmentize (#10087)
OSGeo/gdal: doc/source/programs/ogr2ogr.rst

    topology per feature, in particular for polygon geometries, but not for a
    whole layer.

    The specified value of this option is the tolerance used to merge
    consecutive points of the output geometry using the
    :cpp:func:`OGRGeometry::Simplify` method

>     The unit of the distance is in 
>     georeferenced units of the target vector dataset.
>     This option is applied after the reprojection implied by :option:`-t_srs`
>     Note for the same given tolerance value, actual on the ground
>     east-west tolerance decreases towards the poles.

Reviewer: ```suggestion
    The unit of the distance is in 
    georeferenced units of the source vector dataset.
```

> 

Reviewer 2: The east-west toleranse applies only to some CRS (EPSG:4326, EPSG:3857). In most real world use cases of gdal_footprint this thing does not happen and this texts adds mostly confusion.

Author: Please mention it there.

And mention alternatives if such maps produced are not acceptable.

Thanks.
